lib/Main.js

```diff
@@ -791,9 +788,10 @@ Main.prototype.makeMatrixRoom = function(opts) {
 
 Main.prototype.run = function(port) {
     var bridge = this._bridge;
-
-    const promise = this._gitter.rooms.findAll();
-    console.log(promise);
+    const promiseAllRooms = this._gitter.rooms.findAll().catch((err) => {
```

Comment:
Seems strange to do this here instead of before line 817

Reply:
The reason for this is so we don't repeatedly spam the logs with errors for every single entry in the DB if we fail this one lookup.

Reply:
Ah ok. I think this would be clearer with `async ... await` tbh

Reply:
The reason for *that* was so that we don't block waiting on the request while we also do ``loadDatabases``, ``this.getRoomStore().select``. The hope was that by not doing await we could save a few seconds (or probably a millisecond) this way. I might be totally failing to understand promises.

Reply:
Actually, Node.js's IO is async, which means the two bits of work to (a) get rooms and (b) load databases can be done mostly in parallel. 

Feels slightly like an early optimisation, especially as this only happens once. I think it's probably fine but worth putting a comment to explain whats going on :+1:

Reply:
I think I did that in the latest bit of code :)